### PR TITLE
#7539 - selenium commands reviewed and updated

### PR DESCRIPTION
**Pull request by Vegebutcher**
new: .alertaction and .gettext
.callfunction code needs to be fixed, so I hope to check and update the example before this PR is merged

### CODE REVIEW

File: G1ANT.Addon.Selenium/G1ANT.Addon.Selenium/Commands/SeleniumAlertActionCommand.md [hunk 1]
```diff
@@ -0,0 +1,35 @@
...
+| `errormessage` | [text](G1ANT.Language/G1ANT.Language/Structures/TextStructure.md) | no       |                                                              | A message that will be shown in case the command throws an exception or when a given `timeout` expires, and no `errorjump` argument is specified |
+| `errorresult`  | [variable](G1ANT.Language/G1ANT.Language/Structures/VariableStructure.md) | no       |                                                              | Name of a variable that will store the returned exception. The variable will be of [error](G1ANT.Language/G1ANT.Language/Structures/ErrorStructure.md) structure |
+
+For more information about `if`, `timeout`, `errorcall`, `errorjump`, `errormessage` and `errorresult` arguments, see [Common Arguments](G1ANT.Manual/appendices/common-arguments.md) page.
+
+## Example
+
+In the following example, the robot opens Chrome, runs a Java script, which brings up a confirmation message and after a 2-second delay, confirms this message (equivalent of clicking OK):
```
Comment (WiktoriaPrusik) [inline G1ANT.Addon.Selenium/G1ANT.Addon.Selenium/Commands/SeleniumAlertActionCommand.md]: it's not a Java script. Java and Javascript are different languages and here it is a Javascript script 😄

File: G1ANT.Addon.Selenium/G1ANT.Addon.Selenium/Commands/SeleniumRunScriptCommand.md [hunk 2]
```diff
@@ -1,36 +1,37 @@
...
 ```G1ANT
-selenium.open type ‴firefox‴ url ‴google.com‴
-selenium.runscript script ‴return document.querySelector("body").tagName‴ result ♥result2
-dialog ♥result2
+selenium.open chrome url google.com
+selenium.runscript ‴return document.querySelector("body").tagName‴
+dialog ♥result
 ```
+
+> **Note:** It is crucial to use `return` keyword before Javascript code.
```
Comment (WiktoriaPrusik) [inline G1ANT.Addon.Selenium/G1ANT.Addon.Selenium/Commands/SeleniumRunScriptCommand.md]: Nope. **If you want your Javascript code to return something** then it is crucial to use this keyword. If you want it to just do something (click somewhere for example) then you don't need to add "return" word before your Javascript code using this command :)

File: G1ANT.Addon.Selenium/G1ANT.Addon.Selenium/Commands/SeleniumAlertActionCommand.md (hunk 1 above)
Comment (Vegebutcher) [reply G1ANT.Addon.Selenium/G1ANT.Addon.Selenium/Commands/SeleniumAlertActionCommand.md]: I'm aware of that. One space too many.

Comment (Vegebutcher) [reply G1ANT.Addon.Selenium/G1ANT.Addon.Selenium/Commands/SeleniumAlertActionCommand.md]:
```suggestion
In the following example, the robot opens Chrome, runs a Javascript, which brings up a confirmation message and after a 2-second delay, confirms this message (equivalent of clicking OK):
```

File: G1ANT.Addon.Selenium/G1ANT.Addon.Selenium/Commands/SeleniumRunScriptCommand.md (hunk 2 above)
Comment (Vegebutcher) [reply G1ANT.Addon.Selenium/G1ANT.Addon.Selenium/Commands/SeleniumRunScriptCommand.md]:
```suggestion
> **Note:** It is crucial to use `return` keyword before Javascript code if this code is designed to return something.
```

File: G1ANT.Addon.Selenium/G1ANT.Addon.Selenium/Commands/SeleniumAlertActionCommand.md (hunk 1 above)
Comment (WiktoriaPrusik) [reply G1ANT.Addon.Selenium/G1ANT.Addon.Selenium/Commands/SeleniumAlertActionCommand.md]: Yeah, I saw later that everywhere where there was a Javascript code, you explained it correctly :) But I forgot so I left my first comment unchanged, I know it was just a typo 😄

**Pull request merged**